routes/category.js

```diff
@@ -1,26 +1,25 @@
 const express = require('express');
 const router = express.Router();
-const products = require('../src/data/products');
-const categories = require('../src/data/categories');
+const service = require('../services/dataService');
 
+/*
+ * This method first gets the category ID from the request params.
+ * Then it loops through the combinedProductMap.
+ * It checks if the category id of that particular product is equal to the request param
+ * If matches then push to the response array.
+ */
 router.get('/:ctyId', function(req, res, next) {
-    const ctyId = req.params.ctyId;
-    let currentCategory;
-    let response = [];
-    categories.categories.forEach( category => {
-        if (category.id === ctyId) {
-            currentCategory = category;
-        }
-    });
-    products.products.forEach( product => {
-        if (product.categoryId === ctyId) {
-            response.push({
-                ...product,
-                category: currentCategory
-            });
-        }
-    });
-    res.send(response);
+  service.getCombinedProductMap();
```

Comment:
Can you call `service.getCombinedProductMap()` and assign that to a variable here. 
then use it for your needs below. 
Don't directly call the variables.